lib/model.js

```diff
@@ -353,7 +354,10 @@ Model.prototype.$__handleSave = function(options, callback) {
       }
       this.constructor.exists(where, optionsWithCustomValues)
         .then(documentExists => {
-          const matchedCount = !documentExists ? 0 : 1;
+          const matchedCount = documentExists ? 1 : 0;
```

Comment:
I think I could have coded it even smaller, when I originally wrote that line. 

```suggestion
          const matchedCount = !!documentExists;
```